doc/deploy/android.md

```diff
@@ -1,33 +1,129 @@
-## Deploying to the Google Play Store
-##### Prerequirements
-- You **must** read the [general deploy info](README.md).
-- You **must** have the official PVTrAck signing key. See [the keys doc](android-keys.md) for more info.
-- You **must** have the Android SDK and Android build tools installed (both bundled with Android Studio). We target API level 25 (Nougat, 7.1).
-- You **must** have access to the [Google Play Developer Console](play.google.com/apps/publish) (see @sherson).
-- You *should* have [Android Studio](https://developer.android.com/studio/index.html) installed.
+# Building an Android App to Deploy in Google Play
 
-##### Steps
+## Requirements
+_These instructions assume you are running a Mac. If you're not, the installation
+instructions will be vastly different. You can ask for help or, since you're probably
+running Linux, just translate them like you're probably used to doing._
 
-1. `npm install`
+### General
+- Did you read the [general deploy info](README.md)?
+- Do you have access to the [Google Play Developer Console](play.google.com/apps/publish)?
+ If not, see Adam.
 
-2. Open `config.xml`. On line 2, increment `android-versionCode` with your appropriately chosen new version number.
+### Android Studio
+We use Android Studio to easily manage the Android SDK tools. If you wish, you can attempt
+to download the tools separately, but these instructions will not help you because I found the
+attempt maddening. Plus, Android Studio will keep track of updates for you, which is nice.
+1. Get [Android Studio](https://developer.android.com/studio/)
+1. Use it to download the SDK platforms and tools: 
+  1. Click menu items Android Studio -> Appearance and Behavior -> System Settings -> Android SDK
+  1. In the main window, click "sdk platforms"
+  1. Click 'edit' to choose an sdk location. Follow the prompts. 
+  The default location is fine--keep note of where that is so you can run the tools 
+  from the command line.  Wait for it to download the components.
+  1. Keep waiting
+  1. Check the boxes next to Android 8.0(Oreo) and any others leading up to the latest--you'll see that that's API 
+  level 26, which we target at the moment. 
+  1. Click on the "tools" tab. Make sure "Android SDK Platform Tools", "Android SDK Tools", and
+  "Android SDK Build Tools" are checked. If you don't have access to an Android phone, make sure
+  the Emulator is also installed. Click "apply"
```

Comment:
This is probably a job for someone who has never worked with pvtrack before, but I wonder if the emulator works.